FtpClient/FtpCli/ConsoleParser/ConsoleParser.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Collections.Generic;
+
+namespace ConsoleParser
+{
+  public class ConsoleParser
+  {
+    private List<ConsoleParserCommand> commands = new List<ConsoleParserCommand>();
+
+    public void executeCommand(string command)
+    {
+      string[] args = command.Split(' ');
+      string commandName = args[0];
+      bool found = false;
+      foreach (ConsoleParserCommand c in commands)
+      {
+        if (c.commandName == commandName)
+        {
+          if (args.Length < 2)
```

Comment:
Could there be valid situations where `args.Length == 1`? For example, if we have a command like `ls` it probably wouldn't take additional arguments. Other than that, looks good!